app/bundles/FormBundle/Views/Field/field_helper.php

```diff
@@ -161,6 +161,16 @@
         }
     }
 
+    if ($field['leadField'] && in_array($contactFields[$field['leadField']]['type'], ['datetime', 'date'])) {
```

Comment:
Hi thanks for review.
Code has been updated with requested changes. Please check now.